resources/fervent-ui/Components/cards.css

```diff
@@ -0,0 +1,128 @@
+.card-layout {
+    min-height: auto;
+    background-color: white;
+    box-shadow: rgba(0, 0, 0, 0.15) 1.95px 1.95px 2.6px;
+    display: flex;
+}
+
+.layout-vertical {
+    flex-direction: column;
+    max-width: 20rem;
+}
+
+.layout-horizontal {
+    flex-direction: row;
+    gap: 2rem;
+    max-width: 40rem;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+}
+
+.card-layout:hover {
+    box-shadow: rgba(0, 0, 0, 0.15) 4px 4px 4px;
+    transition: box-shadow 0.2s ease-in-out;
+}
+
+.card-header h4 {
+    font-weight: bold;
+    margin: 0.5rem 1rem;
+    margin-bottom:0.1rem;
```

Comment:
you could use px here for values smaller than 10px

Reply:
Thanks, yes. I will change it to px